crypto/src/main/java/io/warp10/crypto/OrderPreservingBase64.java

```diff
@@ -43,8 +46,16 @@ public class OrderPreservingBase64 {
     for (int i = 0; i < ALPHABET.length; i++) {
       TEBAHPLA[ALPHABET[i]] = (byte) i;
     }
-  }
 
+    // also build a 12bit lut. Encoding per 12bits double speed. (now reaching 30MB/s)
```

Comment:
lut -> look up table. Explain how the table is built so the actual encoding is easier to understand.